lib/src/getDefinitions.ts

```diff
@@ -8,13 +8,17 @@ export const getDefaultConfig = (defaultAppName = "nextjs") => {
     process.env.UNLEASH_SERVER_API_URL ||
     process.env.NEXT_PUBLIC_UNLEASH_SERVER_API_URL;
 
+  const token = process.env.UNLEASH_SERVER_API_TOKEN;
+  const resolvedToken: string | false = (token === 'false' ? false : token || defaultToken);
```

Comment:
Is there a situation where you'd use both token and `UNLEASH_SERVER_INSTANCE_ID`? You can always override `Authentication` with `fetchOptions`

Reply:
> Is there a situation where you'd use both token and UNLEASH_SERVER_INSTANCE_ID?    

I personally don't but based on the documentation I don't think it's inconsistent. 

> You can always override Authentication with fetchOptions

Do you think we would omit the "false" option for the token and let the user overload fetchOptions? In that case it seems less intuitive to me. Besides, it means I would have to set it every time getDefinitions and it wouldn't work globally right?

However, I'll be happy for any variation if it leads to the possibility to omit the token :).